source/replication.rst

```diff
@@ -195,3 +202,82 @@ The *Replica* site will be ready for replication when you apply changes as usual
 
    $ kubectl apply -f deploy/cr.yaml
 
+.. _operator-replication-mcs:
+
+Enabling multi-cluster Services
+--------------------------------------------
+
+Kubernetes `multi-cluster Services (MCS) <https://cloud.google.com/kubernetes-engine/docs/concepts/multi-cluster-services>`_
+is a cross-cluster discovery and invocation of Services. MCS-enabled Services become discoverable and accessible across clusters with a virtual IP address.
+
+This feature allows splitting applications into multiple clusters combined in
+one *fleet*, which can be useful to separate logically standalone parts
+(i.e. stateful and stateless ones), or to address privacy and scalability
+requirements, etc.
+
+Multi-cluster Services should be supported by the cloud provider. It is
+supported `by Google Kubernetes Engine (GKE) <https://cloud.google.com/kubernetes-engine/docs/concepts/multi-cluster-services>`__, and `by Amazon Elastic Kubernetes Service (EKS) <https://aws.amazon.com/blogs/opensource/introducing-the-aws-cloud-map-multicluster-service-controller-for-k8s-for-kubernetes-multicluster-service-discovery/>`__.
+
+Configuring your cluster for multi-cluster Services includes two parts:
+
+* configure MCS with your cloud provider,
+* make needed preparations with the Operator.
+
+To set up MCS for a specific cloud provider you should follow official guides,
+for example ones `from Google Kubernetes Engine (GKE) <https://cloud.google.com/kubernetes-engine/docs/how-to/multi-cluster-services>`__, or `from Amazon Elastic Kubernetes Service (EKS) <https://aws.amazon.com/blogs/opensource/introducing-the-aws-cloud-map-multicluster-service-controller-for-k8s-for-kubernetes-multicluster-service-discovery/>`__.
+
+Setting up the Operator for MCS makes it registering a Services for export to
+other clusters `with a ServiceExport object <https://cloud.google.com/kubernetes-engine/docs/how-to/multi-cluster-services>`_.
+Set the following options in the ``multiCluster`` subsection of the ``deploy/cr.yaml``
```

Comment:
We could add a couple of words about import to 
"and allow cluster to import external services"